newhelm/prompt_formatting.py

```diff
@@ -0,0 +1,17 @@
+from newhelm.prompt import ChatPrompt, ChatRole
+
+
+def format_chat(
```

Comment:
Note that this function may be SUT specific e.g. different SUTs may want to use different strings for roles and message delimiters.

Reply:
Absolutely. The currently infra makes that trivial as each SUT gets to decide to either use this method or write their own.